docs/middle_high_german.rst

```diff
@@ -198,3 +198,17 @@ The ``WordTokenizer`` class takes a string as input and returns a list of tokens
    
    In [4]: word_tokenizer.tokenize(text)
    Out[4]: ['Mīn', 'ougen', 'wurden', 'liebes', 'alsō', 'vol', ',', 'dō', 'ich', 'die', 'minneclīchen', 'ērst', 'gesach', ',', 'daȥ', 'eȥ', 'mir', 'hiute', 'und', 'iemer', 'mē', 'tuot', 'wol', '.']
+
```

Comment:
Is there any specific reason for only including the tnt tagger in the docs? Shouldn't the others also be at least alluded to?

Reply:
No specific reason, I may add other examples.